module/Entra/Microsoft.Entra/Users/Get-EntraUserMembership.ps1

```diff
@@ -71,6 +71,9 @@ function Get-EntraUserMembership {
                 $params["All"] = $PSBoundParameters["All"]
             }
         }
+        elseif (-not $PSBoundParameters.ContainsKey("Top")) {
```

Comment:
This should just be a standalone `if` statement like all of the other parameter checks that precede it. 👍